index.html

```diff
@@ -3,14 +3,20 @@
   <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width">
-    <title>repl.it</title>
+    <title>Madlibs</title>
     <link href="style.css" rel="stylesheet" type="text/css" />
   </head>
   <body>
-    <div class='madLibsEdit'>
-    </div>
-    <div class='madLibsPreview'>
+    <h1 id ="header">Mad <img src="images/fun.jfif" id="img">Libs</h1>
```

Comment:
When creating classes, ids, or attributes, is is best not to put spaces: id ="header" => id="header"
So that, no space between the id and the = sign.